R/fotmob_players.R

```diff
@@ -70,147 +186,21 @@ fotmob_get_match_players <- function(match_ids) {
     stopifnot(length(starters) == 2) ## 2 teams
     stopifnot(length(bench) == 2)
 
-    .clean_positions <- function(p) {
-
-      ## index represents F/M/D/G (1/2/3/4), row index represents player (usually up to 4)
-      player_ids <- as.character(p[["id"]])
-      n <- length(player_ids)
-      ## for some reason jsonlite tries to combine stuff row-wise when it really should just bind things column-wise
-      ps <- p[["stats"]]
-
-      .clean_stats <- function(x) {
-        if(is.null(x)) {
-          return(data.frame(dummy = 1))
-        }
-        xt <- t(x)
```

Comment:
issue was here. the whole `.clean_positions()` was sort of janky, so improved it by re-converting the dataframes to lists using `jsonlite::fromJSON(jsonlite::toJSON())`

Reply:
aside from the change to the internals of `.clean_positions()`, there aren't many actual code changes. mostly just moving things around